src/aat/java/uk/gov/hmcts/cmc/claimstore/tests/functional/citizen/SealedClaimPdfTest.java

```diff
@@ -88,17 +94,8 @@ private Response submitClaim(ClaimData claimData) {
             .post("/claims/" + claimant.getUserDetails().getId());
     }
 
-    private InputStream retrievePdf(String pdfName, String externalId) {
```

Comment:
Removed this method during refactoring exercises, but it's worth keeping. Revert
REQ